CMakeLists.txt

```diff
@@ -105,6 +112,19 @@ include(CmakeFunctionHelper)
 add_subdirectory(src)
 build_library(sycl_blas)
 
+if (WIN32)
+  # On Windows, all symbols must be resolved at link time for DLLs.
+  # The sycl_blas target is just a collection of other objects so
+  # the linked libraries are not going to be propagated to this target.
+  # This requires manual linking against ComputeCpp.lib on Windows.
+  target_link_libraries(sycl_blas PRIVATE ComputeCpp::ComputeCpp)
+
+  set(BUILD_SHARED_LIBS FALSE CACHE BOOL
+    "Force SYCL-BLAS to be built as a static library on Windows"
```

Comment:
This seems a bit strange, the place where the variable is originally set has no description, wouldn't it make more sense to have the description there, where it's not platform dependent, and just have an empty one here?

Reply:
The description is required when forcing the value on the cache so the user can't change it. I think it is better to force the value than required users manually set this to false.